docs/development/doc_guide.rst

```diff
@@ -982,3 +997,27 @@ Narrative documentation guidelines
 
 .. _`docs/common_links.rst`: https://github.com/PlasmaPy/PlasmaPy/blob/main/docs/common_links.rst
 .. |docs/common_links.rst| replace:: :file:`docs/common_links.rst`
+
+.. _`docs/bibliography.bib`: https://github.com/PlasmaPy/PlasmaPy/blob/main/docs/bibliography.bib
+.. |docs/bibliography.bib| replace:: :file:`docs/bibliography.bib`
+
+.. _`IPython.sphinxext.ipython_console_highlighting`: https://ipython.readthedocs.io/en/stable/sphinxext.html?highlight=IPython.sphinxext.ipython_console_highlighting#ipython-sphinx-directive-module
+.. |IPython.sphinxext.ipython_console_highlighting| replace:: ``IPython.sphinxext.ipython_console_highlighting``
+
+.. _`setup.cfg`: https://github.com/PlasmaPy/PlasmaPy/blob/main/setup.cfg
+.. |setup.cfg| replace:: :file:`setup.cfg`
+
+.. _`sphinxcontrib-bibtex`: https://sphinxcontrib-bibtex.readthedocs.io
+.. |sphinxcontrib-bibtex| replace:: ``sphinxcontrib-bibtex``
+
+.. _`sphinx_copybutton`: https://sphinx-copybutton.readthedocs.io
+.. |sphinx_copybutton| replace:: ``sphinx_copybutton``
+
+.. _`sphinx_gallery.load_style`: https://sphinx-gallery.github.io/stable/advanced.html?highlight=load_style#using-only-sphinx-gallery-styles
+.. |sphinx_gallery.load_style| replace:: ``sphinx_gallery.load_style``
+
+.. _`sphinx_changelog`: https://sphinx-changelog.readthedocs.io
+.. |sphinx_changelog| replace:: ``sphinx_changelog``
+
+.. _nbsphinx: https://nbsphinx.readthedocs.io
+.. |nbsphinx| replace:: ``nbsphinx``
```

Comment:
I'm surprised this works!!!!  Essentially a replacement of a replacement, this is super ugly and I'd expected it to break at any minute.  With that said, use single back-ticks for the packages to mimic what it would be like if it was properly set up with `intersphinx`.  Also, could this be added to `common_links.rst` and available for all the documentation?

```suggestion
.. _`docs/bibliography.bib`: https://github.com/PlasmaPy/PlasmaPy/blob/main/docs/bibliography.bib
.. |docs/bibliography.bib| replace:: :file:`docs/bibliography.bib`

.. _`IPython.sphinxext.ipython_console_highlighting`: https://ipython.readthedocs.io/en/stable/sphinxext.html?highlight=IPython.sphinxext.ipython_console_highlighting#ipython-sphinx-directive-module
.. |IPython.sphinxext.ipython_console_highlighting| replace:: ``IPython.sphinxext.ipython_console_highlighting``

.. _`setup.cfg`: https://github.com/PlasmaPy/PlasmaPy/blob/main/setup.cfg
.. |setup.cfg| replace:: :file:`setup.cfg`

.. _`sphinxcontrib-bibtex`: https://sphinxcontrib-bibtex.readthedocs.io
.. |sphinxcontrib-bibtex| replace:: `sphinxcontrib-bibtex`

.. _`sphinx_copybutton`: https://sphinx-copybutton.readthedocs.io
.. |sphinx_copybutton| replace:: `sphinx_copybutton`

.. _`sphinx_gallery.load_style`: https://sphinx-gallery.github.io/stable/advanced.html?highlight=load_style#using-only-sphinx-gallery-styles
.. |sphinx_gallery.load_style| replace:: `sphinx_gallery.load_style`

.. _`sphinx_changelog`: https://sphinx-changelog.readthedocs.io
.. |sphinx_changelog| replace:: `sphinx_changelog`

.. _nbsphinx: https://nbsphinx.readthedocs.io
.. |nbsphinx| replace:: `nbsphinx`
```

Reply:
- [ ] Try it with single backticks and see if a `nitpicky` build leads to errors